lib/buildpack/compile/installers/nodejs_installer.rb

```diff
@@ -24,6 +24,10 @@ class NodeJsInstaller < Installer
     CACHE_DIR = '.node'.freeze
     NPM_COMMAND = 'npm'.freeze
 
+    def self.install_order
```

Comment:
We bumped the `install_order` from the default to make sure that node is installed after the SDK. This is so the node installer can know if the installed SDK uses `project.json` or `*.csproj` and thus knows which of these files to look in to determine whether to install node.